### PR TITLE
Shorten Mauve load test names

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Related to https://github.com/AdoptOpenJDK/openjdk-tests/issues/2109

Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Test : Internal-Grinder/12666

**Comment by Mesbah-Alam** [comment]
This should be delivered together with https://github.com/AdoptOpenJDK/openjdk-systemtest/pull/385

File: system/mauveLoadTest/playlist.xml [hunk 1]
```diff
@@ -38,11 +38,11 @@
 		<platformRequirements>^os.osx</platformRequirements>
 	</test>
 	<test>
-		<testCaseName>MauveSingleThreadLoadTest_HotSpot</testCaseName>
+		<testCaseName>MauveMultiThrdLoad_HS</testCaseName>
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
-		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=MauveSingleThreadLoadTest; \
+		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=MauveMultiThrdLoad; \
```
Comment (llxia) [inline system/mauveLoadTest/playlist.xml:45]: Should this be MauveSingleThreadLoadTest? Not multiple

File: system/mauveLoadTest/playlist.xml [hunk 2]
```diff
@@ -19,11 +19,11 @@
 	</test>
 	<!-- Disabled from OpenJ9 on osx due to : https://github.com/eclipse/openj9/issues/4091 -->
 	<test>
-		<testCaseName>MauveSingleThreadLoadTest_OpenJ9</testCaseName>
+		<testCaseName>MauveMultiThrdLoad_J9</testCaseName>
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
-		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=MauveSingleThreadLoadTest; \
+		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=MauveMultiThrdLoad; \
```
[the remark this answers is not in the document]
Comment (Mesbah-Alam) [reply system/mauveLoadTest/playlist.xml]: Yes indeed! Thanks for spotting this. Corrected.

File: system/mauveLoadTest/playlist.xml (hunk 1 above)
Comment (Mesbah-Alam) [reply system/mauveLoadTest/playlist.xml]: Yes! Corrected.

**Comment by Mesbah-Alam** [comment]
New test with the update to PR: Grinder/12671

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**